src/shrutiDJun20Assignments/ReverseOfInteger.java

```diff
@@ -0,0 +1,24 @@
+package shrutiDJun20Assignments;
+
+public class ReverseOfInteger {
+	
+	void findReverse(int input){
+		int number2 = input;
+		int revnum = 0;
+		while(number2>0){  //235
+			//number=number2%10;   //235%10=5
+			//number2=number2/10;
+			
+			revnum = (revnum*10)+(number2%10);
+			number2 = number2/10;
```

Comment:
You can write proper message what is the input and what is the output of that.

Reply:
changes done as suggested